omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_0/EncounterSearchHandler2_0.java

```diff
@@ -48,8 +48,8 @@ public class EncounterSearchHandler2_0 implements SearchHandler {
 			Collections.singletonList("2.0.* - 9.*"),
 			Collections.singletonList(new SearchQuery.Builder(
 					"Allows you to find Encounter by patient and encounterType (and optionally by from and to date range)")
-					.withRequiredParameters("patient").withOptionalParameters("visit", "encounterType", DATE_FROM, DATE_TO, "order")
-					.build()));
```

Comment:
Can we add a test for this?

Reply:
Sure, on it!

Reply:
I have added `EncounterSearchHandler2_0Test`